nilearn/datasets/description/cobre.rst

```diff
@@ -0,0 +1,123 @@
+    as defined by Power et al. 2012) variables. Each column codes for
+    one variable, starting with the label, and each line has the label of the
+    corresponding subject.
+    :'func': contains list of filenames to functional datasets
+    fmri_szxxxSUBJECT_session1_run1.nii.gz, a 3D+t nifti volume at 3 mm
+    isotropic resolution, in the MNI non-linear 2009a symmetric space
+    (http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin2009).
+    Note that the number of time samples may vary, as some samples have been
+    removed if tagged with excessive motion. See the _extra.mat for more info.
+    :'mat_files': contains list of filenames to .mat files
+    fmri_szxxxSUBJECT_session1_run1_extra.mat, a matlab/octave file for each
+    subject.
+
+    Each .mat file contains the following variables:
+    * confounds: a TxK array. Each row corresponds to a time sample, and each
+      column to one confound that was regressed out from the time series
+      during preprocessing.
+    * labels_confounds: cell of strings. Each entry is the label of a
+      confound that was regressed out from the time series.
+    * mask_suppressed: a T2x1 vector. T2 is the number of time samples in
+      the raw time series (before preprocessing), T2=119. Each entry
+      corresponds to a time sample, and is 1 if the corresponding sample
+      was removed due to excessive motion (or to wait for magnetic
+      equilibrium at the beginning of the series). Samples that were kept
+      are tagged with 0s.
+    * time_frames: a Tx1 vector. Each entry is the time of acquisition
+      (in s) of the corresponding volume.
+
+
+Preprocessing
+-------------
+The datasets were analysed using the NeuroImaging Analysis Kit (NIAK
+https://github.com/SIMEXP/niak) version 0.12.14, under CentOS version 6.3 with
+Octave(http://gnu.octave.org) version 3.8.1 and the Minc toolkit
+(http://www.bic.mni.mcgill.ca/ServicesSoftware/ServicesSoftwareMincToolKit)
+version 0.3.18.
+Each fMRI dataset was corrected for inter-slice difference in acquisition time
+and the parameters of a rigid-body motion were estimated for each time frame.
+Rigid-body motion was estimated within as well as between runs, using the
+median volume of the first run as a target. The median volume of one selected
+fMRI run for each subject was coregistered with a T1 individual scan using
+Minctracc (Collins and Evans, 1998), which was itself non-linearly transformed
+to the Montreal Neurological Institute (MNI) template (Fonov et al., 2011)
+using the CIVET pipeline (Ad-Dabbagh et al., 2006). The MNI  symmetric
+template was generated from the ICBM152 sample of 152 young adults, after 40
+iterations of non-linear coregistration. The rigid-body
+transform, fMRI-to-T1 transform and T1-to-stereotaxic transform were all
+combined, and the functional volumes were resampled in the MNI space at a 3 mm
+isotropic resolution. The "scrubbing" method of (Power et al., 2012), was used
```

Comment:
Yes, but as Pierre said in the issue that it will take time and we decided to proceed with this version.